pygeo/DVConstraints.py

```diff
@@ -3897,6 +4578,7 @@ def __init__(self, name, axis, origin, p0, v1, v2, lower, upper, scale,
 
         # Now embed the coordinates and origin into DVGeo
         # with the name provided:
+        # TODO this is duplicating a DVGeo pointset (same as the surface which originally created the constraint)
```

Comment:
If this is not done please add an issue on github and reference it in the comment